src/latexify/codegen/function_codegen.py

```diff
@@ -162,21 +162,47 @@ def visit_Match(self, node: ast.Match) -> str:
             if i < len(node.cases) - 1:
```

Comment:
According to the [grammar specs](https://docs.python.org/3/reference/grammar.html) we always have at least one `case`, so maybe we can split this for loop into two parts?

```python
for case in node.cases[:-1]:
    ...

case = node.cases[-1]
...
```

This way we can get rid of `enumerate`, as well as the `if i < len(node.cases) - 1` check in each iteration of the loop, which imo would make it a bit cleaner since we do something different for everything but the last element.

We would then also have to move the case validity check into a separate, "private" method.

Reply:
It would be okay to apply it after this PR.